moderation_queue/templates/moderation_queue/_photo-upload-form.html

```diff
@@ -26,7 +26,7 @@
     If you still want to upload another photo of {{ name }}, first select an image from your computer:
     {% endblocktrans %}
     {% else %}
-    {% trans "First, select an image from your computer:" %}
+    {% trans "First, select an image from your computer (maximum size: 50Mb):" %}
```

Comment:
I _think_ technically this should be "50 MB" – lowercase "b" is bits, uppercase "B" is bytes, and the most writing styleguides (and the SI guidelines) recommend putting a space between the number and the unit, unless the unit is a special symbol like a degree sign or a prime mark.